### PR TITLE
Fixes #9917:  Requires Java8 (jdk8) for Rudder 4.1

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9917

### CODE REVIEW

File: rudder-jetty/SPECS/rudder-jetty.spec [hunk 1]
```diff
@@ -81,15 +81,11 @@ BuildArch: noarch
...
 %if 0%{?rhel} && 0%{?rhel} > 6
-Requires: jre >= 1.7
+Requires: jre >= 1.8
 %endif
 
 %if 0%{?rhel} && 0%{?rhel} == 6
-Requires: java-1.7.0-openjdk
-%endif
-
-%if 0%{!?rhel}
```
Comment (amousset) [inline rudder-jetty/SPECS/rudder-jetty.spec]: why removing this?

Comment (peckpeck) [inline rudder-jetty/SPECS/rudder-jetty.spec]: the require of jre >= 1.6 was completely useless

File: rudder-jetty/SPECS/rudder-jetty.spec [hunk 2]
```diff
@@ -81,15 +81,11 @@ BuildArch: noarch
...
-Requires: jre >= 1.6
+Requires: java-1.8.0-openjdk
```
Comment (amousset) [inline rudder-jetty/SPECS/rudder-jetty.spec]: why note jre >= 8?

Comment (peckpeck) [inline rudder-jetty/SPECS/rudder-jetty.spec]: becasue on rhel6 it is called java-1.8.0-openjdk like it was for java 7

**Pull request merged**